manifest.json.template

```diff
@@ -105,9 +105,9 @@
       "artifact": "apiml-sample-extension-*.zip"
     },
     "org.zowe.configmgr": {
-      "version": "0.3.0-V2.X-RC",
+      "version": "0.4.0-V2.X-RC",
```

Comment:
you have configmgr here, but are missing configmgr rexx.
"org.zowe.configmgr-rexx": {
  "version": "0.4.0-V2.X-RC",
  "artifact": "configmgr_rexx-0.4.0-2022090143.pax",
  "repository": "libs-snapshot-local",
}